P3-Project_Planning/README.md

```diff
@@ -10,10 +10,18 @@ For this assignment, you'll create some initial plans for your project.
 
 Include at least 3 wireframes here, adding images or files to your repository as necessary. Format them as a bulleted/unordered list with links to the files.
 
+* Wire Frames
+    *![wire frame photo 1]
+    (C:\Users\Zach\Pictures\WireFramePhotos)
```

Comment:
Looks like you're linking to your local directory. Make sure you drag and drop the photos into the directory for this assignment and provide relative links to them.